framework/src/problems/FEProblemBase.C

```diff
@@ -6758,6 +6763,8 @@ FEProblemBase::computeBounds(NonlinearImplicitSystem & libmesh_dbg_var(sys),
   _lower.swap(lower);
   _upper.swap(upper);
 
+  //BWS: In all other cases where this is called, it's within a try/catch to strip
```

Comment:
This is a note to self that I will remove when I have resolved this outstanding question.